src/components/TextField.vue

```diff
@@ -360,6 +408,12 @@ export default {
     &:active {
       @apply border-red-300;
     }
+
+    &:focus,
+    &:active {
+      box-shadow: 0 0 0 4px rgb(231, 126, 143, 0.3);
```

Comment:
Should this be a tailwind config change too?

Reply:
Added it to the  tailwind config as `error-focus`